webpages/settings/components/addon-body.html

```diff
@@ -253,6 +267,10 @@
   [disabled] {
     cursor: initial !important;
   }
+  .addon-split-button {
+    align-self: flex-start;
```

Comment:
Not really important, but I think the `align-self` should stay specific to the table setting dropdown to make this class more reusable.

Reply:
Removing it makes the table setting dropdown appear hovered when the mouse is to the right of it:

<img width="320" height="197" alt="split_button_align" src="https://github.com/user-attachments/assets/6a8bcf07-4013-4a81-9ffb-465b4c1e52ee" />